File: sample/python/main_set_node_sample.py
import time
# voxelammingパッケージからVoxelammingクラスをインポートします
from voxelamming_local import Voxelamming

# Voxelammingアプリに表示されている部屋名を指定してください
room_name = "1000"
# Voxelammingクラスのインスタンスを生成します
voxelamming = Voxelamming(room_name)

# ボクセルの設定を行います
voxelamming.set_box_size(0.5)
voxelamming.set_build_interval(0.01)

# ボクセルを配置するため、位置と色を設定します
for i in range(10):
    voxelamming.create_box(-1, i, 0, r=0, g=1, b=1)
    voxelamming.create_box(0, i, 0, r=1, g=0, b=0)
    voxelamming.create_box(1, i, 0, r=1, g=1, b=0)
    voxelamming.create_box(2, i, 0, r=0, g=1, b=1)

for i in range(5):
    voxelamming.remove_box(0, i * 2 + 1, 0)
    voxelamming.remove_box(1, i * 2, 0)

for i in range(5):
    voxelamming.transform(-25 + i * 10, 0, 0, pitch=0, yaw=0, roll=0)
    # ボクセルデータをアプリに送信します。
    voxelamming.send_data()
    time.sleep(0.1)

File: sample/javaScript/mainSetNodeSample.mjs
// import { Voxelamming } from 'voxelamming';
import Voxelamming from './voxelamming.js';  // test

const roomName = '1000';
const voxelamming = new Voxelamming(roomName);

voxelamming.setBoxSize(0.5);
voxelamming.setBuildInterval(0.01);

for (let i = 0; i < 10; i++) {
  voxelamming.createBox(-1, i, 0, 0, 1, 1);
  voxelamming.createBox(0, i, 0, 1, 0, 0);
  voxelamming.createBox(1, i, 0, 1, 1, 0);
  voxelamming.createBox(2, i, 0, 0, 1, 1);
}

for (let i = 0; i < 5; i++) {
  voxelamming.removeBox(0, i * 2, 0);
  voxelamming.removeBox(1, i * 2 + 1, 0);
}

for (let i = 0; i < 5; i++) {
  voxelamming.transform(-25 + i * 10, 0, 0, 0, 0, 0);
  await voxelamming.sendData();
  await voxelamming.sleepSecond(0.1)
  console.log('send data done')
}
